proto/tendermint/liquidity/v1beta1/liquidity.proto

```diff
@@ -350,7 +350,7 @@ message WithdrawMsgState {
     MsgWithdrawWithinBatch msg = 6 [(gogoproto.moretags) = "yaml:\"msg\""];
 }
 
-// SwapMsgState defines the state of swap message that contains state information as it is processed in the next batch(s)
+// SwapMsgState defines the state of the swap message that contains state information as the message is processed in the next batch or batches.
```

Comment:
the text is grammatically correct with the article "the" and "a" present as I suggested throughout, thanks!